app/models/saved_claim/caregivers_assistance_claim.rb

```diff
@@ -24,7 +24,13 @@ def process_attachments!
   def to_pdf(filename = nil, **)
     # We never save the claim, so we don't have an id to provide for the filename.
     # Instead we'll create a filename with this format "10-10cg_{uuid}"
-    PdfFill::Filler.fill_form(self, filename || guid, **)
+    name = filename || guid
+    PdfFill::Filler.fill_form(self, name, **)
+  rescue => e
+    Rails.logger.error("Failed to generate PDF: #{e.message}")
+    PersonalInformationLog.create!(data: { form: parsed_form, file_name: name },
```

Comment:
I would advise caution around logging to this PersonalInformationLog table. Is this cleaned up? I'm assuming the parsed form contains PII?

Reply:
Yeah, I agree with the concern. I know we have a `DeleteOldPiiLogsJob`  that runs daily and deletes logs more than 2 weeks old, so these should get automatically cleaned up I believe. So these do get cleaned up. It looks like we have had about a dozen errors that I would expect to be caught here over the last two weeks. I'm hoping to be able to use the data in there to see if we have a parsing issue with some kind of characters or something similar. My goal for this would be we find something here to make it more resilient so we have fewer going forward, and potentially just remove the `PersonalInformationLog` create once we have a better understanding the of the issue. If you still have concerns with it at the moment, I'm fine to just add the Rails logger for now and see what that gives us.

Reply:
I know the `PersonalInformationLog`s for other forms include the form data, too, cf:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/sidekiq/hca/ezr_submission_job.rb#L23

https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/caregivers_assistance_claims_controller.rb#30

and when errors arise, we frequently use them to gather PII like contact info for affected Veterans. Is it a concern there, too?